lib/src/test_harness.dart

```diff
@@ -20,12 +20,12 @@ AssetId idFromString(String s) {
 /// A helper package provider that has files stored in memory, also wraps
 /// [Barback] to simply our tests.
 class TestHelper implements PackageProvider {
-
   /// Maps from an asset string identifier of the form 'package|path' to the
   /// file contents.
   final Map<String, String> files;
   final Iterable<String> packages;
-  final List<String> messages;
+  final List messages;
```

Comment:
personal preference, but I like `List<dynamic>` over `List` -- the former signals to readers that you have explicitly opted out of type checking.